README.md

```diff
@@ -59,6 +64,13 @@ ansible-galaxy collection install -r meta/collection-requirements.yml
 
 ### Defined in `defaults/main.yml`
 
+#### `ha_cluster_get_info`
```

Comment:
I thought initially that  it makes sense to rename this to `ha_cluster_facts` for consistency with other roles. Although this functionality is a little more than just getting the current state, it exports the configuration in a format that can be used to run the role.
Maybe rename this to `ha_cluster_export_configuration`, I think it's more in the face naming, if we export the configuration it implies that we can import it with the same role, which is the case here.
@richm WDYT?

Reply:
Ok - sounds good - please rename to `ha_cluster_export_configuration` and then I think we're good to go.